.github/pull_request_template.md

```diff
@@ -19,7 +19,7 @@ Add a brief summary of your PR here, between one sentence and two paragraphs
 
 By placing an x in the boxes I certify that I have:
 
-- [ ] Ran `cargo test --all` locally if this modified any rs files.
+- [ ] Ran `cargo test --all --features "empty"` locally if this modified any rs files.
```

Comment:
I don't think there's much sense to that. The reason we included this was to run tests which require a graphics driver, but `empty` has no advantage over CI. I'm not sure if we still have tests which require a graphics driver, and if we do, we need to automate it anyway. So, maybe this line should just be removed?

Reply:
`"empty"` would be a good sanity check before pushing, but ya CI would guard against a broken `master`.

Also, yeap the `"graphics"` tests have been removed.

Reply:
I'm of the opinion that everyone should be running the tests on all their code. I'm not sure we need the CI noise of a bunch of failed PR's just because we removed this line. It doesn't do much harm imo. Thoughts @torkleyy @azriel91

Reply:
I prefer people do run the tests as well